cmd/agent/flow_run.go

```diff
@@ -214,6 +219,24 @@ func (fr *flowRun) Run(configFile string) error {
 		defer func() { _ = srv.Shutdown(ctx) }()
 	}
 
+	// Report usage of enabled components
+	if !fr.disableReporting {
+		infos := f.ComponentInfos()
+		components := []string{}
+		for _, info := range infos {
+			components = append(components, info.Name)
+		}
```

Comment:
There may be many components with the same name (such as more than one `discovery.kubernetes` component). We'll probably want to only put each component name once.